python/CMakeLists.txt

```
@@ -0,0 +1 @@
+install(DIRECTORY edm4hep DESTINATION python)
```

```suggestion
install(DIRECTORY edm4hep DESTINATION python
  REGEX .*\\.in EXCLUDE   # don't install configuration input files
  PATTERN __pycache__ EXCLUDE  # don't install python caches
  )
```
Final comment to make sure to only install what we want